daemon/execdriver/driver.go

```diff
@@ -102,6 +102,7 @@ type Resources struct {
 	Memory     int64  `json:"memory"`
 	MemorySwap int64  `json:"memory_swap"`
 	CpuShares  int64  `json:"cpu_shares"`
+	CpuQuota   int64  `json:"cpu_quota"`
```

Comment:
This should really go into hostConfig instead of config, these other ones are likely to be moved as well here:  #10298
Having it in config means it gets embedded into the image config as well.